### ISSUE TITLE
Config isn't parsed correctly

### ISSUE THREAD

**Issue by JohJohan** [editorconfig-checker/editorconfig-checker#198]
When I run the tool on my projects I see allot of errors for example wrong indent while the indent is correct. When I check my editor config file i have:

```
[*]
indent_size=4

[*.{yaml, yml}]
indent_size=2
```

Now when I change it to:

```
[*]
indent_size=4

[*.{yaml,yml}]
indent_size=2
```

I don't have errors I don't think spaces are parsed correctly when parsing the config file. While spaces are allowed.

**Comment by mstruebing** [comment]
Hello, thanks for your issue but I think this would be a bug in https://github.com/editorconfig/editorconfig-core-go could you create an issue there?

I'm not sure what is allowed and what is not, but when looking at the examples here: https://editorconfig.org/
they are without spaces.

**Comment by JohJohan** [comment]
You are right, I also don't know what is allowed but I use it in combination with PHP storm editor and there indenting with spaces in config works

**Comment by mstruebing** [comment]
I would read that differently as `[ *.{yaml,yml} ]` this would be allowed.
Regardless, we do not parse the `.editorconfig` ourselves but rather use https://github.com/editorconfig/editorconfig-core-go for this.
So if this is an issue it should be raised there.
